pkg/git/gitlab/gitlab_helper.go

```diff
@@ -92,7 +92,7 @@ func GetBaseUrl(repoUrl string) (string, error) {
 func refineGitHostingServiceError(response *http.Response, originErr error) error {
 	// go-gitlab APIs do not return a http.Response object if the error is not related to an HTTP request.
 	if response == nil {
-		if strings.Contains(originErr.Error(), "The provided authorization grant is invalid") || strings.Contains(originErr.Error(), "401 Unauthorized") {
+		if strings.Contains(originErr.Error(), "The provided authorization grant is invalid") || strings.Contains(originErr.Error(), "401 Unauthorized") || strings.Contains(originErr.Error(), "invalid header field value") {
```

Comment:
Could you explain why `"invalid header field value"` is the unauthorized error? Will it be always so?

Reply:
changed it to Private-Token